templates/data_collector/msp.html.twig

```diff
@@ -12,12 +12,23 @@
                 <span class="sf-toolbar-status">{{ value|slice(0,30) }}... {% if value|length > 30 %}+{{ value|slice(30)|length }} chars{% endif %}</span>
             </div>
         {% endfor %}
+        <div class="sf-toolbar-info-piece">
+            <b><label for="session_id">Session ID:</label> <input id="session_id" type="number" value="1" style="width: 60px" /></b>
+            <button id="simulations_start" type="button" onClick="(function(){
```

Comment:
anonymoud ajax call to /{session}/_profiler/msp_start_simulations.
Not using twig function path(...) since I cannot inject the session id beforehand . It is only available at run-time.

I could however generate the path using argument session = 999 for instance, and do string replace on 999 to the actual id... not necessarily better...